pkg/service/contract/consumer/consumer.go

```diff
@@ -0,0 +1,302 @@
+// Package consumer is a package for consumer driven contract testing
+package consumer
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/fatih/color"
+	"github.com/olekukonko/tablewriter"
+	"go.keploy.io/server/v2/config"
+	schemaMatcher "go.keploy.io/server/v2/pkg/matcher/schema"
+	"go.keploy.io/server/v2/utils"
+
+	"go.keploy.io/server/v2/pkg/models"
+	"go.uber.org/zap"
+)
+
+const IDENTIFYMODE = 0
```

Comment:
You can put this in models and make a proper enum maybe call that enum as SchemaMatchMode or some other good name you can ask chatgpt.